docs/pages/management/register-services/register-services-auth-service.mdx

```diff
@@ -81,16 +104,15 @@ On your local machine, use the `tctl` tool to generate a new token. In the
 following example, a new token is created with a TTL of five minutes:
 
 ```code
-# Generate a short-lived invite token for a new node:
-$ export INVITE_TOKEN=$(tctl nodes add --ttl=5m --roles=node | grep "invite token:" | grep -Eo "[0-9a-z]{32}")
+$ export INVITE_TOKEN=$(tctl nodes add --ttl=5m --roles=node | grep "join token:" | grep -Eo "[0-9a-z]{32}")
```

Comment:
Maybe we should use `tctl tokens add` instead. This was originally written when SSH nodes were the only type of supported resource.